discord/src/main/groovy/com/asm/tavern/discord/discord/audio/SongsCommandHandler.groovy

```diff
@@ -51,10 +54,12 @@ class SongsCommandHandler implements CommandHandler {
 					StringBuilder builder = new StringBuilder()
 					songs.forEach({ song ->
 						currentCategory = song.category
-						if(song.uri.getScheme() == "http" || song.uri.getScheme() == "https") {
+						try {
+							new URL(song.uri.toString())
```

Comment:
Songs with no scheme in songs command will not hyperlink this way or the previous way but it is cleaner looking